### DISCUSSION TITLE
Github Education - Global Campus unable to verify my student ID

### DISCUSSION THREAD

**Discussion by MengchengHan** [community/community#52500] [GitHub Education]
Hi there, 

I' ve been trying to upload my student condition to the Global Campus 5 times and it keeps saying "Rejected". I' ve uploaded my student ID, my scholar insurance document with no success. Also, I' ve tried to contact Github support team and they gave me an automatic answer, like if you are having troubles with Github Education check this link, if your problem hasn't been resolved yet please go check this link. All those links I' ve visited and those are the documentation and some basic tutorials to "upload" files.

Also, whenever I "upload" some document, there is no upload button, there is only take a photo option, so awkard holding my document and trying to take a picture with the webcam.   

This is the rejection message: 

Unfortunately, we weren't able to approve your educational discount request for [@MengchengHan](https://github.com/MengchengHan).

What happened?
Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

**Comment by foysal619** [comment]
I'm sorry to hear that you're having trouble getting your student condition approved on the GitHub Global Campus platform. Here are a few suggestions that may help:

Double-check the documentation: Make sure you've carefully reviewed the documentation provided by GitHub to understand what is required to successfully verify your student status. The link provided in the rejection message may be a good place to start.

Verify your documents: Check that your student ID and scholar insurance documents meet the requirements specified by GitHub. Make sure they are clear and legible, and that they include all the necessary information.

Contact GitHub support again: If you've reviewed the documentation and verified your documents, but are still having trouble, you may need to reach out to GitHub support again. Be sure to provide a detailed explanation of the issue you're experiencing, and include any relevant screenshots or documents that may help them understand the problem.

Consider alternative verification methods: If you continue to experience difficulty with the current verification process, you may want to consider alternative verification methods. For example, some universities offer a service that allows students to verify their enrollment status through a third-party provider. Check with your school to see if this is an option.

Seek assistance from a teacher or administrator: If you're still having trouble, consider reaching out to a teacher or administrator at your school for assistance. They may be able to provide additional guidance or help you navigate the verification process more effectively.

I hope these suggestions are helpful in resolving the issue you're experiencing with the GitHub Global Campus platform.

**Comment by samus-aran** [comment]
Hi @MengchengHan 

We've updated the details to your applied school. You should be able to re-apply. Let us know how you get on 👍
